layout_padded/layout_padded.md

```diff
@@ -1343,60 +1397,108 @@ then `Extents::static_extent(Extents::rank() - 1)` is a multiple of `LayoutRight
+[2.2]{.pnum} Let _`is-layout-left-padded-mapping-of`_ be
+the exposition-only variable template defined as follows,
+
+```c++
+template<class Mapping>
+constexpr bool @_is-layout-left-padded-mapping-of_@ = // exposition only
+  /* @_see-below_@ */;
+```
+
+where _`is-layout-left-padded-mapping-of`_`<Mapping>` is `true` if and only if `Mapping` denotes a specialization of `layout_left_padded<S>::mapping` for some value `S` of type `size_t`.
+
+<!--
+A typical implementation.
 
 ```c++
 template<class Layout>
 struct @_is-layout-left-padded_@ : // exposition only
   false_type {};
 
-template<size_t padding_stride>
-struct @_is-layout-left-padded_@<layout_left_padded<padding_stride>> : // exposition only
+template<size_t padding_value>
+struct @_is-layout-left-padded_@<layout_left_padded<padding_value>> : // exposition only
   true_type {};
 
 template<class Mapping>
-constexpr bool @_is-layout-left-padded-mapping-of_@ // exposition only
+constexpr bool @_is-layout-left-padded-mapping-of_@ = // exposition only
   @_is-layout-left-padded_@<typename Mapping::layout_type>::value;
 ```
+-->
+
+[2.3]{.pnum} Let _`is-layout-right-padded-mapping-of`_ be
+the exposition-only variable template defined as follows,
+
+```c++
+template<class Mapping>
+constexpr bool @_is-layout-right-padded-mapping-of_@ = // exposition only
+  /* @_see-below_@ */;
+```
 
-* [2.3]{.pnum} let _`is-layout-right-padded-mapping-of`_ be
-the exposition-only variable template defined as follows.
+where _`is-layout-right-padded-mapping-of`_`<Mapping>` is `true` if and only if `Mapping` denotes a specialization of `layout_right_padded<S>::mapping` for some value `S` of type `size_t`.
+
+<!--
+A typical implementation.
 
 ```c++
 template<class Layout>
 struct @_is-layout-right-padded_@ : // exposition only
   false_type {};
 
-template<size_t padding_stride>
-struct @_is-layout-right-padded_@<layout_right_padded<padding_stride>> : // exposition only
+template<size_t padding_value>
+struct @_is-layout-right-padded_@<layout_right_padded<padding_value>> : // exposition only
   true_type {};
 
 template<class Mapping>
-constexpr bool @_is-layout-right-padded-mapping-of_@ // exposition only
+constexpr bool @_is-layout-right-padded-mapping-of_@ = // exposition only
   @_is-layout-right-padded_@<typename Mapping::layout_type>::value;
 ```
+-->
 
 > In Section � *[mdspan.layout.stride.cons]*,
-> in paragraph 7 (Remarks for the constructor
-> `layout_stride::mapping(const StridedLayoutMapping&)`),
-> add the following two lines immediately below
-> _`is-mapping-of`_`<layout_right, LayoutStrideMapping> ||`
-> and above
-> _`is-mapping-of`_`<layout_stride, LayoutStrideMapping> ||`:
+> change paragraph 9 (*Remarks* for the constructor
+> `layout_stride::mapping(const StridedLayoutMapping&)`)
+> by adding the two lines shown in green to the code expression
+> that follows the "is equivalent to" phrase.
+
+[9]{.pnum} *Remarks*: The expression inside `explicit` is equivalent to:
```

Comment:
2024/01/31: Add more diff context so we can see to what constructor this applies.  Show the constructor to which it applies.

Reply:
Do this by adding a code block with the signature of the constructor.

Reply:
Done in PR #441.